pkg/addons/default/kube_proxy.go

```diff
@@ -172,13 +155,8 @@ func addArm64NodeSelector(daemonSet *v1.DaemonSet, archLabel string) error {
 	return fmt.Errorf("NodeAffinity not configured on kube-proxy. Either manually update the proxy deployment, or switch to Managed Addons")
 }
 
-func getLatestKubeProxyImage(input AddonInput, greaterThanOrEqualTo1_18 bool) (string, error) {
+func getLatestKubeProxyImage(input AddonInput) (string, error) {
 	defaultClusterVersion := generateImageVersionFromClusterVersion(input.ControlPlaneVersion)
-	// EKS Addons API only works for 1.18 and above
-	if !greaterThanOrEqualTo1_18 {
-		return defaultClusterVersion, nil
-	}
-
```

Comment:
don't we need this check for 1.19 and above?

Reply:
Anything below is now no longer supported.

Reply:
> Anything below is now no longer supported.

Yeah, so don't we need this check to return early `if !greaterThanOrEqualTo1_19` ? 🤔

Reply:
sorry was out at hospital, thanks for replying @Skarlso ✨ 

Yeah as @Skarlso says since no clusters older than 1.18 can exist, we don't need to check anymore. Its a guarantee that the cluster will be compatible

Reply:
cool 👍🏻